### DISCUSSION TITLE
wrong error message without libcamera support

### DISCUSSION THREAD

**Discussion by wrohdewald** [Motion-Project/motionplus#90] [Enhancements]
### Did you read the guide?

Yes

### What is the base version number of MotionPlus being used?

0.1.x

### What was the install method?

Built from source code

### What is base architecture?

ARM-32bit

### What is the distro being used?

Raspbian

### Disto version number

bullseye

### Camera/Sound type(s) being used?

PI camera via libcam

### Describe the issue/problem and steps to reproduce

Without libcamera support compiled in (I forgot to install libcamera-dev):

`mlp_init_cam_start: Failed to start camera.`

which is not very helpful. IMHO this is a bug. A correct error message might be

`mlp_init_cam_start: Failed to start camera. Reason: libcamera support not compiled in.`

Even better: without libcamera support, treat parameter libcam_device as unsuppported

### Relevant MotionPlus log output (at log_level 8)

```shell
see above
```

**Comment by Mr-Dave** [comment]
Moving to enhancements.
